### PR TITLE
Announcing the NodeCopter Lounge at CascadiaJS

### PR DESCRIPTION
**Pull request by voodootikigod**
New event announcement for a more relaxed, during conference lounge style event to encourage interest and excitement leading up to the 2013 festivities.

### CODE REVIEW

File: pages/2012/cascadia/nov-89.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+<p><strong>Awesome, Where Do I Sign Up?</strong></p>
+
+<p>Unlike the bigger NodeCopter.js events, the lounge events are designed to attract passerby interest during a conference (in this case <a href="http://cascadiajs.com/">CascadiaJS</a>) and let you try out the NodeCopters before attending the full-fledged hackathon event. There will be many different projects being shown and constructed during the conference time so stop by at any point. If a group of no more than three find a project they want to pursue, they will be given a dedicated copter to develop with. In total, the goal is to have fun... WITH FLYING ROBOTS!</p>
+
+<p>The NodeCopter Lounge will be open during the normal hours of CascadiaJS and do require a CascadiaJS Attendee Ticket to participate, but nothing further.</p>
+
+<p><strong>Final thoughts</strong></p>
+
+<p>During the Nodecopter Dublin event very recently, Felix Geisendörfer wrote a library to control the Parrot AR Drone 2.0 with javascript using node.js. He also just wrote a <a href="https://github.com/felixge/node-cross-compiler">tool to install node.js ON THE DRONE</a>.</p>
```
Comment (felixge) [inline pages/2012/cascadia/nov-89.html:44]: That was written before the Berlin event.

**Comment by felixge** [comment]
LGTM. I made the edit from my comment. A few more things before merging:
- The city is seatle, no cascadia, right? If so, the folder name should change.
- The date format should be `nov-8-9` instead of `nov-89` (in the filename).
- Any reason you used HTML over Markdown? I'm fine with it as long as you're happy maintaining it, just asking.
- Y U HATEZ 80 CHAR LINE BREAKS? (no biggie, but makes github code views/diffs kinda useless as they will require horizontal scrolling)

**Comment by voodootikigod** [comment]
I just copied the Brighton one ;). That should answer your format q's. cool on date handling.

**Comment by rmehner** [comment]
Fixed the filename thing.

**Comment by felixge** [comment]
@rmehner thanks. I think the folder also needs to be renamed to `seattle`. Can you do that real quick as well?

**Pull request merged**